tests/test_files.py

```diff
@@ -304,3 +304,41 @@ def test_PlateRecording__get_well_indices__returns_sorted_set(
 ):
```

Comment:
I can't make a comment directly on the H5 file...but isn't this supposed to be v0.1.1, not v0.1? tests/h5/v0.1/MA20001100__2020_07_15_172203__A4.h5

Reply:
That file is actually v0.1. I included it just in case we ever wanted to do any testing with them since it's the only one I've seen. The file I used for v0.1.1 is from 7/22.